make fails when module is Python only

On the latest master build, make fails with the following error when trying to build Python-only OOT modules

[ 42%] Built target pygen_python_6b7f4
make[2]: *** No rule to make target '../python/bindings/gnuradio-dragon', needed by 'python/bindings/extracted_docstrings.json'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:350: python/bindings/CMakeFiles/extracted_docstrings.dir/all] Error 2
make: *** [Makefile:141: all] Error 2

This error persists until a C++ block is added to the module. It seems like cmake doesn't have a way to handle not having any pybind11 bindings.

--- a/gr-utils/modtool/templates/gr-newmod/python/__init__.py
+++ b/gr-utils/modtool/templates/gr-newmod/python/__init__.py
@@ -14,11 +14,10 @@ import os
 
 # import pybind11 generated symbols into the howto namespace
 try:
+    # this might fail if the module is python-only
     from .howto_python import *
 except ImportError:
-    dirname, filename = os.path.split(os.path.abspath(__file__))
-    __path__.append(os.path.join(dirname, "bindings"))
-    from .howto_python import *
+    pass
 
 # import any pure python here
 #

--- a/gr-utils/modtool/templates/gr-newmod/python/bindings/CMakeLists.txt
+++ b/gr-utils/modtool/templates/gr-newmod/python/bindings/CMakeLists.txt
@@ -5,6 +5,17 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 #
 
+########################################################################
+# Check if there is C++ code at all
+########################################################################
+if(NOT howto_sources)
+    MESSAGE(STATUS "No C++ sources... skipping python bindings")
+    return()
+endif(NOT howto_sources)
+
+########################################################################
+# Check for pygccxml
+########################################################################
 GR_PYTHON_CHECK_MODULE_RAW(
     "pygccxml"
     "import pygccxml"

